### ISSUE TITLE
Startup fails when an old zlib1.dll is in PATH

### ISSUE THREAD

**Issue by lovetox** [quodlibet/quodlibet#2817]
Hi, 

I have this problem on Windows 10, but i guess it applys to other Windows versions also
quodlibet 4.0.2

If a old zlib1.dll is in PATH GTK will use whatever is in PATH over what quodlibet ships itself

here is one dll to test with 
[zlib1.zip](https://github.com/quodlibet/quodlibet/files/1849287/zlib1.zip)

**Comment by lazka** [comment]
There is a known bug in mingw python where it prefers libraries in the system directories: https://github.com/Alexpux/MINGW-packages/issues/3381

Where did you find the old zlib1.dll?

**Comment by lovetox** [comment]
To be honest, im developing on a xmpp client Gajim, we took most of the packaging code for Windows from your work here at quodlibet.

A user reported this problem to us, he has as it seems a very old installation of GnuPG.

I also looked at Inkscape as they also use MSYS and GTK, but they are somehow not affected by this.

i tried setting the path to mingw32/bin on Windows, in MSYS, with python, but it seems this all does not work.

the bug with the openssl dlls was easy to fix as we just had to move the dlls as it was suggested in the issue.

As you seem more knowledgeable about MSYS/GTK in general, i thought i bring this to your attention this way, i hope this is not a problem.
